core/os/os_linux.odin

```diff
@@ -406,6 +443,56 @@ access :: inline proc(path: string, mask: int) -> (bool, Errno) {
 	return true, ERROR_NONE;
 }
 
+getpwnam :: proc(name: string) -> ^passwd {
+	cstr := strings.clone_to_cstring(name);
```

Comment:
Might be good to use the temp allocator for these things.
We do that elsewhere, for instance.

You can still defer-delete it though which will prevent it from leaking if it's too big.
https://github.com/odin-lang/Odin/blob/674aeffee4483aff5030ec40082ecd77da47f7a0/core/runtime/default_allocators.odin#L106-L114

Reply:
Well, yeah, I was considering it, but some of the other functions in this file that I didn't add didn't do this. The only one that did was like set_current_directory or something (I'll have to check on this).

Reply:
Also, are you sure about the delete with the temp allocator, because last time somebody told me this, it crashed my program, lol